docs/tools/flow-cli/flix.md

```diff
@@ -96,8 +96,85 @@ pub fun main(): String {
 }
 ```
 
+
+### Cadence Doc Pragma
+It's recommended to use pragma to set the metadata for the script or transaction. More information on [Cadence Doc Pragma FLIP](https://github.com/onflow/flips/blob/main/application/20230406-interaction-template-cadence-doc.md) 
```

Comment:
Can we add one line about what pragma is here for people that don't know? Also, missing a period at the end of the paragraph.